modules/monitoring/application/views/scripts/event/show.phtml

```diff
@@ -15,6 +15,12 @@ echo $object instanceof Service
     ? '<h2>' . $this->translate('Current Service State') . '</h2>' . $this->render('partials/object/service-header.phtml')
     : '<h2>' . $this->translate('Current Host State') . '</h2>' . $this->render('partials/object/host-header.phtml');
 ?>
+
+<?php
+foreach ($extensionsHtml as $extensionHtml) {
+    echo $extensionHtml;
+}
+?>
```

Comment:
It's fine that you've moved this upwards but now it's part of the controls section. Didn't you mention to place it right before the "Event Details" table? Please do so :wink: